### PR TITLE
Issue 1178 helm secrets integration

### PR DESCRIPTION
**Pull request by RoryShively**
addresses issue 1178
https://github.com/GoogleContainerTools/skaffold/issues/1178

### CODE REVIEW

**Comment by nkubala** [comment]
also, please sign the CLA :)

File: pkg/skaffold/deploy/helm.go [hunk 1]
```diff
@@ -265,7 +269,12 @@ func (h *HelmDeployer) deployRelease(ctx context.Context, out io.Writer, r lates
 	}
 	args = append(args, setOpts...)
 
-	helmErr := h.helm(ctx, out, args...)
+	useSecrets := false
+	if r.UseHelmSecrets {
+		useSecrets = true
+	}
+
+	helmErr := h.helm(ctx, out, useSecrets, args...)
```
[the remark this answers is not in the document]
Comment (RoryShively) [reply pkg/skaffold/deploy/helm.go]: Yes thats an obvious improvement. Will fix

**Comment by RoryShively** [comment]
> So there's good news and bad news.
> 
> 👍 The good news is that everyone that needs to sign a CLA (the pull request submitter and all commit authors) have done so. Everything is all good there.
> 
> 😕 The bad news is that it appears that one or more commits were authored or co-authored by someone other than the pull request submitter. We need to confirm that all authors are ok with their commits being contributed to this project. Please have them confirm that here in the pull request.
> 
> _Note to project maintainer: This is a terminal state, meaning the `cla/google` commit status will not change from this state. It's up to you to confirm consent of all the commit author(s), set the `cla` label to `yes` (if enabled on your project), and then merge this pull request when appropriate._

Can somebody explain what this means and how I can address it?

**Comment by nkubala** [comment]
@RoryShively sorry, looks like you need a rebase here :( once that's done and Kokoro passes we can get this merged

**Comment by RoryShively** [comment]
@nkubala I rebased but I don't know what Kokoro is. How does it work? Whats the process to update it's status

**Comment by RoryShively** [comment]
@nkubala @balopat I don't know how to fix this cla issue. I accidentally used another account to push my commits but have signed the cla for both of my accounts. Does that need to be manually overridden?

**Pull request merged**